orderbook/openapi.yml

```diff
@@ -118,20 +118,28 @@ paths:
         404:
           description: Order was not found
     delete:
-      summary: Deletes order from order service,
+      summary: Deletes order from order service
```

Comment:
ubernit: Maybe call it invalidate or cancel rather than delete here and in the description below (since we are not actually deleting it).